src/test/suite/compile.test.ts

```diff
@@ -0,0 +1,106 @@
+import * as assert from 'assert';
+import { after, before, describe, it, beforeEach } from 'mocha';
+import * as path from 'path';
+import * as fs from 'fs-extra';
+
+import { Uri, window, commands, workspace, TextDocument, TextEditor, Position, SnippetString } from 'vscode';
+
+const logPath: string = '/tmp/vscode-inmanta.log';
+const workspaceUri: Uri = Uri.file(path.resolve(__dirname, '../../../src/test/workspace'));
+const libsPath: string = path.resolve(workspaceUri.fsPath, 'libs');
+const envPath: string = path.resolve(workspaceUri.fsPath, '.env');
```

Comment:
This will probably have to change once you merge in master

Reply:
Why? Does the place where the environment is created changes?

Reply:
Indeed, see #30

Reply:
From https://github.com/inmanta/vscode-inmanta/pull/33#discussion_r515010393 I guess it does, I will take a look.

Reply:
I don't see where/when/how the environment folder is created, how is the location decided? Is there an environment variable set somewhere?

Reply:
Found it, I didn't merge yet

Reply:
Is it really the same venv as for `compilerVenv` though? `.env` still gets created for me, no matter the path I set for the venv.

Reply:
Hmm, it shouldn't, right @andraskvr?

Reply:
That's right.
I checked out this branch and the tests indeed fail on this point for me.
Note that the compiler venv change affected both the language server and the extension. So you have to update both for it to work (`pip install -U server/` or something similar, and `vsce package`, then Extensions -> Install from VSIX for the extension part)

Reply:
I "fixed it" here https://github.com/inmanta/vscode-inmanta/pull/33/commits/920a3825eb58b9276f22b9d15b0df69f25eb4e2b, but it doesn't work for me.  
A part of it is that I don't know how to get the default path (inside the tests I mean, I would need to access the context of the extension, but I couldn't figure it out yet), which will be in my case: `/home/guillaume/.config/Code/User/workspaceStorage/6607cd65b9884be7da6505e5121d968a/Inmanta.inmanta/.env-ls-compiler`.
Another part is that that `.env` folder keeps getting created for me anyway.

Reply:
Actually, I simply removed all checks on the venv, as this as to be verified in this issue right? https://github.com/inmanta/vscode-inmanta/issues/39